code/modules/antagonists/heretic/heretic_antag.dm

```diff
@@ -559,6 +561,10 @@
 /datum/antagonist/heretic/proc/get_knowledge(wanted)
 	return researched_knowledge[wanted]
 
+/datum/antagonist/heretic/proc/increase_rust_strength(strength)
+	if(rust_strength < strength)
+		rust_strength = strength
```

Comment:
- `increase_rust_strength` is a weird name for this, maybe go with like `set_rust_strength`. 
- Either check that `strength` is a number, or have strength be a default value, for sanity
```dm
if(!isnum(strength))
   return // Or crash
```